sleepylyze/nrem.py

```diff
@@ -167,7 +164,7 @@ def lowpass_raw(self, lowpass_freq=25, lowpass_order=4):
         
 
         # set dataframe columns
-        data_lowpass.columns = pd.MultiIndex.from_arrays([channels, np.repeat(('raw_lowpass'), len(channels))],names=['Channel','datatype'])
+        data_lowpass.columns = pd.MultiIndex.from_arrays([self.channels, np.repeat(('raw_lowpass'), len(self.channels))],names=['Channel','datatype'])
```

Comment:
this will run faster if the object attribute (self.channels) is accessed once and set to a local variable rather than accessed multiple times